test/integration/targets/ansible-galaxy-collection/tasks/pinned_pre_releases_in_deptree.yml

```diff
@@ -19,7 +19,7 @@
     command: >-
       ansible-galaxy collection install
       meta_ns_with_transitive_wildcard_dep.meta_name_with_transitive_wildcard_dep
-      rc_meta_ns_with_transitive_dev_dep.rc_meta_name_with_transitive_dev_dep:=2.4.5-rc5
+      rc_meta_ns_with_transitive_dev_dep.rc_meta_name_with_transitive_dev_dep:==2.4.5-rc5
```

Comment:
This change was to have the `ansible-galaxy-collection` integration tests pass. According to the [docs](https://docs.ansible.com/ansible/latest/collections_guide/collections_installing.html#installing-an-older-version-of-a-collection), `==` should be used to install pre-release candidates, but I'd presume this has worked up to this point.

Reply:
I think both single and double equals has historically been functional in both roles and collections. If there's a test, this means that we expected it to work.